tidoop-mr-lib-api/conf/tidoop-mr-lib-api.json

```diff
@@ -1,3 +1,11 @@
 {
-  "port": 6060
+  "port": 6060,
```

Comment:
Since this a JSON file, I think I should move to a .js file in order to add those comments :S